addon/services/document.js

```diff
@@ -0,0 +1,74 @@
+import Service, { inject as service } from "@ember/service";
+import fetch from "fetch";
+
+export default class DocumentService extends Service {
```

Comment:
Maybe rename this to `documents? Since this service handles the `documents` endpoint i think this is more appropriate than `document`

Reply:
I used the singular as we do with models.

Reply:
Yes but the model name represents a model type or a single record. And the service is a api to interact with all Documents. Also i find it a bit confusing if both the service and the model are named the same since they dont represent the same api / type.

Reply:
Okey

Reply:
Done